packages/headless/src/features/facets/range-facets/date-facet-set/date-facet-actions.ts

```diff
@@ -161,7 +215,7 @@ export interface ToggleSelectDateFacetValueActionCreatorPayload {
   /**
    * The target date facet value.
    */
-  selection: DateFacetValue;
+  selection: DateFacetApiValue;
```

Comment:
Why this change? From the perspective of a developer using Headless, they would only see `DateFacetValue` when using the controller. I think `DateFacetApiValue` is an internal concept that should not be exposed?